packages/flutter/lib/src/services/raw_keyboard_macos.dart

```diff
@@ -0,0 +1,235 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+
+import 'keyboard_key.dart';
+import 'keyboard_maps.dart';
+import 'raw_keyboard.dart';
+
+/// Platform-specific key event data for macOS.
+///
+/// This object contains information about key events obtained from macOS's
+/// `NSEvent` interface.
+///
+/// See also:
+///
+///  * [RawKeyboard], which uses this interface to expose key data.
+class RawKetEventDataMacOs extends RawKeyEventData {
+  /// Creates a key event data structure specific for macOS.
+  ///
+  /// The [characters], [charactersIgnoringModifiers], and [modifiers], arguments
+  /// must not be null.
+  const RawKetEventDataMacOs(
+      {this.characters = '',
+      this.charactersIgnoringModifiers = '',
+      this.keyCode = 0,
+      this.modifiers = 0})
+      : assert(characters != null),
+        assert(charactersIgnoringModifiers != null),
+        assert(keyCode != null),
+        assert(modifiers != null);
+
+  /// Documentation extracted from Apple's NSEvent doc:
```

Comment:
You don't really mean that, right?  These don't appear to be copied from that documentation (which is good: we can't do that).

I would rather this were a  "See also" section with a link to Apple's NSEvent doc, added to each of the properties than come from the NSEvent.

```suggestion
  /// See also:
  ///
  ///   * [Apple's NSEvent documentation](https://developer.apple.com/documentation/appkit/nsevent?language=objc)
```

Reply:
Thanks! Added a link to each property